supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -100,5 +100,67 @@ Text for note.
 ====
 ----
 
-// [[product-names-versions-ref]]
-// == Product names and version references
+[product-names-versions-ref]]
+== Product names and version references
+
+Whenever you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product, avoid using hard-coded references and use attributes instead.
+Only use hard-coded version references if the version that you are referring to in a particular case never changes.
+
+=== Attribute file
+
+Define attributes for product name and product version and store them in a dedicated attributes file for each set of product documentation.
+For examples of where you can store the shared attributes file inside your documentation repository, see the link:https://github.com/redhat-documentation/modular-docs/tree/mod-doc-repo-example[Example modular documentation repository].
+Include the attributes file at the beginning of the `master.adoc` files of all titles in your documentation set:
+
+.master.adoc
```

Comment:
Can you update this title to be consistent with our other examples? (with Example in front)